src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -1105,6 +1154,26 @@ static string mapPath(SourceReferenceResolver? referenceResolver, SyntaxTree tre
             {
                 return referenceResolver?.NormalizePath(tree.FilePath, baseFilePath: null) ?? tree.FilePath;
             }
+
+            // Caller must free the returned builder.
+            ArrayBuilder<string> getNamespaceNames()
```

Comment:
>ArrayBuilder [](http://example.com/codeflow?start=12&length=12)

Consider passing in the symbol or `ContainingNamespace` and making this a `static` method.